src/RockPhysics/RockPhysics.jl

```diff
@@ -0,0 +1,10 @@
+export RockPhysics
+
+module RockPhysics
+
+    using JUDI:find_water_bottom
```

Comment:
That's really quite the heavy dependency for just that one function, would implement here.